src/apps/explorer/pages/Solver/index.tsx

```diff
@@ -95,40 +109,15 @@ const Solver: React.FC = () => {
   useEffect(() => {
     setQuery('')
     setPageOffset(0)
-  }, [networkId, setPageOffset, setQuery])
-
-  tableState['hasNextPage'] = tableState.pageOffset + tableState.pageSize < tableValues.length
-  tableState['totalResults'] = tableValues.length
-
-  useEffect(() => {
-    if (query.length) {
-      setPageOffset(0)
-    }
-  }, [query, setPageOffset])
-
-  useEffect(() => {
-    setQuery('')
-    setPageOffset(0)
-  }, [networkId, setPageOffset, setQuery])
+  }, [networkId, setPageOffset, setQuery, tabViewSelected])
```

Comment:
Is the useEffect intentionally denpendent on `tabViewSelected` although it's not used in it?

Reply:
This is intentional since we need to reset the search bar query  when changing between tabs